build.gradle

```diff
@@ -1,5 +1,6 @@
 buildscript {
   repositories {
+    mavenCentral()
```

Comment:
jcenter seems to be unreliable

Reply:
Maybe there was an issue, but I wouldn't add another repo because of an intermittent failure.